momento-sdk/src/main/java/momento/sdk/responses/storage/data/GetResponse.java

```diff
@@ -0,0 +1,77 @@
+package momento.sdk.responses.storage.data;
```

Comment:
it feels a tiny bit weird that these have `.data` in the package name when the cache ones do not

Reply:
I can remove the extra level of nesting. It's too bad there is no way to roll up the names as in other languages.

Reply:
Done in [fbcda96](https://github.com/momentohq/client-sdk-java/pull/358/commits/fbcda961d89ea39425b8d35cea5791018d2ced17)